src/index.html

```diff
@@ -18,8 +18,8 @@ <h2 class="sub-heading">Recent Scores</h2>
     </div>
     <div class="score-input">
       <h2 class="sub-heading2">Add your score</h2>
-      <input class ="name" type="text" placeholder="Your name">
-      <input class="score" type="text" placeholder="Your score">
+      <input class ="name" type="text" placeholder="Your name" required>
+      <input class="score" type="text" placeholder="Your score" required>
```

Comment:
- [ OPTIONAL ] I highly recommend making the type for the score number instead of text to not let the user submit a text instead of a number so it would be great if you fix it ✨✅⭐ I am asking as optional because this is not part of the requirements in this milestone ⭐✨🙏🏻